Composer/packages/server/src/utility/project.ts

```diff
@@ -7,17 +7,28 @@ import { remove } from 'fs-extra';
 import formatMessage from 'format-message';
 import { UserIdentity } from '@botframework-composer/types';
 
+import { isElectron } from '../utility/isElectron';
 import { ExtensionContext } from '../models/extension/extensionContext';
 import { LocationRef } from '../models/bot/interface';
 import settings from '../settings';
 import log from '../logger';
 import AssetService from '../services/asset';
 import { BotProject } from '../models/bot/botProject';
 import { BackgroundProcessManager } from '../services/backgroundProcessManager';
-import { runDialogMerge } from '../workers/dialogMerge.worker';
 
 import { Path } from './path';
 
+let workerPath = '@bfc/server-workers/lib/dialogMerge.worker';
```

Comment:
preference to extract this into some utility and abstract the electron logic on L26. this sets up an easy extension in a hosted scenario where we may want some additional flexibility into where workers are held.

This would reduce to the following on L30
```typescript
const { runYeomanTemplatePipeline } = require(getWorkerPath('dialogMerge'));
```